inherit-python-string/lower_string.py

```diff
@@ -0,0 +1,16 @@
+class LowerString(str):
+    def __init__(self, string):
+        super().__init__(string.lower())
```

Comment:
I think you can either remove this example or add a comment that it doesn't work. My vote would be for removing it.

Reply:
HI @lpozo, you didn’t make any changes here. I’m fine with keeping it if it’s your deliberate choice. I just want to double check that you didn’t forget about it 😉

Reply:
Ohh, I missed this comment. I'll fix it now. Thanks!